cypress/integration/search.spec.js

```diff
@@ -1,30 +1,49 @@
 context('Searching', () => {
-
-  describe('I want to know what a Cree word means in English', () => {
+  
+  describe('I want to see search results showing dynamically while I type', () => {
     // https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/issues/120
-    it('should search for an exact lemma', () => {
+    it('should search for first minos, then minosis', () => {
       cy.visit('/')
-      cy.get('[data-cy=search]')
-        .type('minos')
+      cy.search('minos')
+        .searchResultsContain('cat')
 
-      cy.get('[data-cy=search-results]')
-        .should('contain', 'cat')
+      // makes it minos+is = minosis
+      cy.search('is')
+        .searchResultsContain('kitten')
     })
+  })
```

Comment:
This is a new test that directly tests for typing on the search bar

Reply:
This is SOOOO NICE 🤩  🤩  🤩